libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRoutes.kt

```diff
@@ -0,0 +1,106 @@
+package com.mapbox.navigation.core.fasterroute
+
+import com.mapbox.navigation.base.ExperimentalMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.directions.session.RoutesObserver
+import com.mapbox.navigation.core.directions.session.RoutesUpdatedResult
+import com.mapbox.navigation.utils.internal.logE
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.launch
+import java.util.concurrent.CopyOnWriteArraySet
+
+@ExperimentalMapboxNavigationAPI
+fun MapboxNavigation.createFasterRoutes(
```

Comment:
I didn't want to make part of mapbox navigation because I thought it will make more complex to copy it and apply to the Nav SDK 2.8. But not I think that it can even simplify everything. I will try.